demos/tests/args.py

```diff
@@ -52,6 +52,15 @@ def resolve(self, context):
         return str(context.dl_dir / context.model_info[self.name]["subdirectory"] / self.precision / (self.name + '.xml'))
 
 
+class ModelDirArg:
+    def __init__(self, model_name, file_name):
+        self.model_name = model_name
+        self.file_name = file_name
+
+    def resolve(self, context):
+        return str(context.dl_dir / context.model_info[self.model_name]["subdirectory"] / self.file_name)
```

Comment:
You need to update the model downloading logic so that an argument that is an instance of this class triggers the downloading of the corresponding model.

Reply:
updated